README.md

```diff
@@ -136,3 +136,28 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+Creating tags that correspond with releases and release candidates in [`ansible/ansible`](https://github.com/ansible/ansible)
+
+First, install the additional tagging dependencies
+
+```bash
+pip install gitpython typer
```

Comment:
```suggestion
python -m pip install -r hacking/tagger/requirements.txt
```

Reply:
There's a requirements file in the `tagger` folder that you can use. It might also be an idea to suggest creating a venv and installing into that to avoid polluting the user space.

Reply:
Yeah, this was the bit I'd had questions about if I'd done right. I'd initially followed [the manual setup guide](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally), and then looked at the `nox` setup as well, but neither got the needed requirements. 

I'd written this part *assuming* that the reader had done one of those at this point (which I shouldn't do). 

Is there something cleaner than this? If not, I'll update it to at least suggest creating a venv.

Reply:
Not quite sure what you mean by cleaner. I suppose you don't really need the `python -m` portion and can trim it down to just `pip install`.

The venv bit can just be a comment in the codeblock too so it's discrete. I'd imagine anyone doing this would have a venv already so just a hint is all that's needed really.

Reply:
Do you think that + the updated info above makes this clear enough, and has sufficient reminder about the venv?

Reply:
Yes, I think that's fine now. Thank you.